addons/project/views/project_views.xml

```diff
@@ -18,7 +18,7 @@
                <search string="Tasks">
                     <field name="name" string="Task"/>
                     <field name="tag_ids"/>
-                    <field name="user_ids" filter_domain="[('user_ids', 'ilike', self), ('user_ids.active', 'in', [True, False])]"/>
+                    <field name="user_ids" filter_domain="[('user_ids.name', 'ilike', self), ('user_ids.active', 'in', [True, False])]"/>
```

Comment:
Can you explain the issue in the commit message?

I would like to see:
- steps to reproduce the issue
- Actual behavior
- Expected behavior

Without this, I cannot know what is the bug that you fixed.

Reply:
> Can you explain the issue in the commit message?
> 
> I would like to see:
> 
> * steps to reproduce the issue
> * Actual behavior
> * Expected behavior
> 
> Without this, I cannot know what is the bug that you fixed.

@xavierbol I updated the commit message as you can see. thank you

Reply:
Thanks